app/components/field_component.rb

```diff
@@ -7,7 +7,7 @@ def initialize(name, form:, type: 'text', css_class: [], **options)
     @type = type
     @form = form
     @css_class = Array(css_class)
-    @options = options
+    @options = options.except(:form)
```

Comment:
What's this for?

Reply:
I was having trouble with this `FieldComponent` because It was rendered with a `form=` attribute in the HTML and that attribute was breaking the input by letting the form submit empty data.

However I think that we need to find a better solution, it seems that in Ruby 3 the **options parameter now catches all the parameters of the method.

<img width="1237" alt="Screen Shot 2021-02-25 at 09 31 28" src="https://user-images.githubusercontent.com/24940330/109153864-3d9f4500-774c-11eb-8ce4-4573ecec5ebc.png">

<img width="1058" alt="Screen Shot 2021-02-25 at 09 31 43" src="https://user-images.githubusercontent.com/24940330/109153896-48f27080-774c-11eb-916f-bf5f835aa4c7.png">

Reply:
@brunvez I think that I've found a better way, check my last commit 🙌

Reply:
@juanmferreira93 I think I liked it better the other way. What I wanted on that component was to replicate the API used beneath so that the component resembles an HTML tag. I'd agree to add the `form` as the same first param but I think `:options` makes the API un-elegant